### PR TITLE
Update python and packages

### PR DESCRIPTION
**Pull request by JamesGardiner**

### CODE REVIEW

**Review by petechd — APPROVED** [review]
Api and scripts working as expected👍

File: Pipfile [hunk 1]
```diff
@@ -5,7 +5,7 @@ name = "pypi"
 
 [packages]
 Flask = "*"
-attrs = "==19.1.0"
+attrs = "==19.2.0"
 jsonpointer = "*"
 eq-translations = {editable = true,git = "https://github.com/ONSDigital/eq-translations.git",ref = "v2.0.0"}
```
Comment (MebinAbraham) [inline Pipfile:10]: Can't quite remember why we never upgraded this?

**Review by MebinAbraham — DISMISSED** [review]

File: Pipfile (hunk 1 above)
Comment (JamesGardiner) [reply Pipfile:10]: If we don't upgrade it, linting fails with an invalid kwarg argument error for the equality `eq` kwarg

Comment (MebinAbraham) [reply Pipfile:10]: I meant the eQ translations package.

Comment (ajmaddaford) [reply Pipfile:10]: We should remove this and see if it works as it's not a direct requirement. The reason for it being there was probably to fix a transitive dependency issue that has been resolved.

Comment (JamesGardiner) [reply Pipfile:10]: I'll give it a try

**Review by MebinAbraham — APPROVED** [review]

**Pull request merged**